src/EnergyPlus/UnitarySystem.cc

```diff
@@ -3778,12 +3778,8 @@ namespace UnitarySystems {
         this->m_HeatOutAirVolFlow = input_data.heating_oa_flow_rate;
         this->m_NoCoolHeatOutAirVolFlow = input_data.no_load_oa_flow_rate;
 
-        // UnitarySystemType_Num isn't used anywhere
```

Comment:
Since this wasn't being used, replaced it with two new enums (one for zone equip and one for airloop). @rraustad